specs/commandline/verify.md

```diff
@@ -0,0 +1,99 @@
+# notation verify
+
+## Description
+
+Use `notation verify` command to verify signatures on an artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. The digest of the supplied artifact is returned upon successful verification. It is recommended that this digest reference be used to pull the artifact subsequently, as registry tags may be mutable, and a tag reference can point to a different artifact that what was verified.
+
+## Outline
+
+```text
+Verify signatures associated with the artifact.
+
+Usage:
+  notation verify [flags] <reference>
```

Comment:
We should explain here and in the sign command that < reference > support can be a tag or a digest.  When with a tag, we do a default tag to digest translation and we only sign/verify the digest, not the tag itself.

Reply:
I will create a new PR to address this